source/userspace/display_management.rst

```diff
@@ -1,89 +1,221 @@
+* The third argument goes hand-in-hand with the :code:`layout` option.
+
+It looks like the :code:`layout` is where the black-magic happens. Let's have a closer look at :code:`layout` s!
+
+Layouts
+-------
+
+Layouts are nothing more than rules that specify the format / number of lines / fonts we are going to use on the screen.
+They are easy to remember once you understand that:
+
+* :code:`b` stands for **bold**.
+* :code:`n` stands for **normal**.
+* :code:`p` stands for **picture**.
+* :code:`paging` means that if the data doesn't fit on screen, user will be able to navigate through multiple screens to see the data (e.g a public key).
+* And the number of letters used stands for the **number of lines**!
+
+Note that **all combinations of letters are not possible**. For example :code:`paging` only exists with :code:`bnnn`. :code:`nnnn` only exists on NanoX, etc...
+
+Now that you know all that, here's a table with the **most commonly used layouts**:
+
+.. list-table:: Most commonly used layouts
+	:widths: 5 55 40
+	:header-rows: 1
+	
+	* - Denomination
+	  - Comment
+	  - Usage
+	* - :code:`bn`
+	  - Bold font for the first line, normal font for the second line.
+	  - :code:`bn, {"BoldLine", "NormalLine"}`
+	* - :code:`pb`
+	  - Picture for the first line, bold font for the second line.
+	  - :code:`pb, {&RefToPicture, "BoldLine"}`
+	* - :code:`pnn`
+	  - Picture for the first line, normal font for the second line and third line.
+	  - :code:`pnn, {&RefToPicture, "NormalLine1", "NormalLine2"}`
+	* - :code:`bnnn_paging`
+	  - Bold first line, normal fonts for the other lines. If the data to be displayed doesn't fit on a single screen, the user will be able to navigate through different screens to see the whole text.
+	  - :code:`bnnn_paging, {.title = "BoldLine", .text = "NormalLine"}`
+
+
+.. |nanos_bnnn_paging| image:: images/nanos/nanos_address.png
+   :scale: 100%
+.. |nanox_bnnn_paging| image:: images/nanox/nanox_address.png
+   :scale: 100%
+.. |nanos_bn| image:: images/nanos/nanos_amount.png
+   :scale: 100%
+.. |nanox_bn| image:: images/nanox/nanox_amount.png
+   :scale: 100%
+.. |nanos_pb| image:: images/nanos/nanos_approve.png
+   :scale: 100%
+.. |nanox_pb| image:: images/nanox/nanox_approve.png
+   :scale: 100%  
+.. |nanos_pnn| image:: images/nanos/nanos_boilerplate.png
+	:scale: 100%
+.. |nanox_pnn| image:: images/nanox/nanox_boilerplate.png
+   :scale: 100%  
+
+And here's a table that compares how those layouts are displayed on a Nano S and on a Nano X!
+
+Notice that the **Nano X can fit up to 4 lines**, whereas the **Nano S can only fit 2**!
+
+.. list-table:: Comparing end results on NanoS and NanoX
+	:widths: 10 40 40
+	:header-rows: 1
+
+	* - LAYOUT
+	  - NANOS
+	  - NANOX
+	* - :code:`pb`
+	  - |nanos_pb|
+	  - |nanox_pb|
+	* - :code:`bn`
+	  - |nanos_bn|
+	  - |nanox_bn|
+	* - :code:`pnn`
+	  - |nanos_pnn|
+	  - |nanox_pnn|
+	* - :code:`bnnn_paging`
+	  - |nanos_bnnn_paging|
```

Comment:
can I haz the 2 other nano S screens for paging, to make even more obvious ?

Reply:
which ones are they ? I though we could only have `bnnn_paging` for some reason ^^

Reply:
I mean, the 2 other screenshots for bnnn_paging on nano S (currently only the first (1/3) screen is shown, HOW CONFUSING)

Reply:
Ohhhhhh ok... but then the question is how do we fit those in the table ? ^^'
And also it could confuse users even more because they could end up thinking only the nano S has this capability... hmmm unless we take a screenshot of a string that's long enough so that the paging activates on both nano S and nano X ?

Reply:
merge pics in a single pic :galaxy-brain:
you're right, a take a string that fits in 2 Nano X screens :+1:

Reply:
https://github.com/LedgerHQ/ledger-dev-doc/pull/55/commits/5390fef9007abb2d744f97a2a4ce2abf4c1b6dae